test/k8sT/manifests/guestbook_deployment.yaml

```diff
@@ -78,17 +79,19 @@ spec:
 kind: Service
 apiVersion: v1
 metadata:
-  name: redis-slave
+  name: redis-follower
   labels:
-    k8s-app.guestbook: redis
-    role: slave
+    app: redis
+    role: follower
+    tier: backend
 spec:
   ports:
   - port: 6379
     targetPort: redis-server
   selector:
-    k8s-app.guestbook: redis
-    role: slave
+    app: redis
+    role: follower
+    tier: backend
```

Comment:
Same as for the v1.8 backport: the frontend image below will need to be updated to match the Docker image version used in master:
```diff
-         image: gcr.io/google-samples/gb-frontend:v4
+         image: gcr.io/google-samples/gb-frontend:v6
```

Reply:
Thanks a lot for the hint, I just repushed